src/components/investment-cards/UniswapInvestmentCard.js

```diff
@@ -14,72 +14,77 @@ import InvestmentCardPill from './InvestmentCardPill';
 
 const UniswapInvestmentCardHeight = 114;
 
+const gradientColors = [
+  colors.uniswapInvestmentCards.startGradient,
+  colors.uniswapInvestmentCards.endGradient,
+];
+
 const AssetLabel = withProps({
   color: colors.alpha(colors.blueGreyDark, 0.6),
   lineHeight: 'tight',
   size: 'smedium',
 })(Text);
 
-const enhance = compose(
-  withAccountSettings,
-  pure,
-  withHandlers({
-    onPressContainer: ({ item, onPress }) => () => {
-      if (onPress) {
-        onPress(item);
-      }
-    },
-  })
-);
+const UniswapInvestmentCard = ({ isCollapsible, item, onPress, ...props }) => {
+  const { nativeCurrency } = useAccountSettings();
+  const { openInvestmentCards } = useOpenInvestmentCards();
+
+  const {
+    ethBalance,
+    tokenBalance,
+    tokenName,
+    tokenSymbol,
+    totalBalanceAmount,
+    totalNativeDisplay,
+    uniqueId,
+  } = item;
+
+  const handlePressContainer = useCallback(() => {
+    if (onPress) {
+      onPress(item);
+    }
+  }, [item, onPress]);
 
-const UniswapInvestmentCard = enhance(
-  ({
-    isCollapsible,
-    item: {
-      ethBalance,
-      tokenBalance,
-      tokenName,
+  const headerProps = useMemo(
+    () => ({
+      color: colors.dark,
+      emoji: 'unicorn',
+      isCollapsible,
+      title: `ETH • ${tokenSymbol}`,
+      titleColor: '#E540F1',
```

Comment:
I've noticed we're using this color also in `SwapDetailsState.js`, should we move it to the colors file or replace it by one of the existing ones? cc: @christianbaroni